lucene/core/src/java/org/apache/lucene/search/LRUQueryCache.java

```diff
@@ -656,13 +713,20 @@ public long ramBytesUsed() {
     // threads when IndexSearcher is created with threads
     private final AtomicBoolean used;
 
+    private Executor executor;
```

Comment:
`final`?

Reply:
I did not add that due to the way the constructors use each other -- for the default case, the executor is null.

Reply:
Hmm can we duplicate a little constructor code and then make it `final`?